packages/upload/test/keyboard-navigation.common.js

```diff
@@ -31,61 +31,108 @@ describe('keyboard navigation', () => {
     uploadElement.files = [FAKE_FILE];
 
     await nextRender();
-
-    fileElement = uploadElement.querySelector('vaadin-upload-file');
+    uploadButton = document.querySelector('vaadin-button[slot=add-button]');
+    fileElements = uploadElement.querySelector('vaadin-upload-file');
```

Comment:
This change is confusing as the variable is `fileElements` which indicates "multiple files" in which case it should use `querySelectorAll`. Consider either keeping `fileElement` or using `querySelectorAll`.

Reply:
Yeah, that makes sense, the reason I changed it to fileElementS was to have one variable for when there multiple files as well. But it makes sense to use `querySelectorAll` because now I can also change the file focus test to use `fileElements[0]` which is more consistent with the other uses. Fixed :+1: